chain/client/src/view_client.rs

```diff
@@ -358,7 +358,10 @@ impl ViewClientActor {
         }
 
         let head = self.chain.head().map_err(|e| TxStatusError::ChainError(e))?;
-        let target_shard_id = self.runtime_adapter.account_id_to_shard_id(&signer_account_id);
+        // TODO: get_tx may not work properly after re-sharding for transactions processed
+        //       by previous shards, fix this
```

Comment:
Please create an issue if you do not plan to fix it in the same PR

Reply:
actually this is not an issue any more, I will remove the comment